packages/vite/src/utils/generator-utils.ts

```diff
@@ -696,3 +735,230 @@ export async function handleUnknownExecutors() {
     `);
   }
 }
+
+export function checkForBuildOptionsInViteConfig(
```

Comment:
Can we name this `ensureBuildOptionsInViteConfig`? "check" is implying it doesn't change anything.

Reply:
Sure!!